tests/tests.js

```diff
@@ -1,45 +1,141 @@
+  beforeEach,
+  describe,
+  it
+} = require('mocha')
 const employee = require('./employee')
 const products = require('./products')
 const pricing = require('../pricing')
 
-describe('calculateProductPrice', () => {
-  it('returns the price for a voluntary life product for a single employee', () => {
-    const selectedOptions = {
-      familyMembersToCover: ['ee'],
-      coverageLevel: [{ role: 'ee', coverage: 125000 }],
-    }
-    const price = pricing.calculateProductPrice(products.voluntaryLife, employee, selectedOptions)
+chai.use(sinonChai)
+const {
+  expect
+} = chai
+
+describe('formatPrice', () => {
+  it('returns a truncated price with two decimal places not rounded when given a number with multiple places', () => {
+    const x = 15.1566663
+    const formattedPrice = pricing.formatPrice(15.1566663)
 
-    expect(price).to.equal(39.37)
+    expect(formattedPrice).to.equal(15.15)
   })
+})
 
-  it('returns the price for a voluntary life product for an employee with a spouse', () => {
-    const selectedOptions = {
-      familyMembersToCover: ['ee', 'sp'],
-      coverageLevel: [
-        { role: 'ee', coverage: 200000 },
-        { role: 'sp', coverage: 75000 },
-      ],
+describe('commuterTrain', () => {
+  it('calculates the employee fare on the commuter train, minus the contribution from the employer', () => {
+    const commuterTrain = {
+      price: 84.75 - (products.employerContribution.contribution)
     }
-    const price = pricing.calculateProductPrice(products.voluntaryLife, employee, selectedOptions)
 
-    expect(price).to.equal(71.09)
+    expect(commuterTrain).to.equal(9.75)
   })
+})
 
-  it('returns the price for a disability product for an employee', () => {
-    const selectedOptions = {
-      familyMembersToCover: ['ee']
+describe('commuterParking', () => {
+  it('calculates the fee payable by the employee for parking, short of the employer contribution', () => {
+    const commuterParking = {
+      price: 250 - (products.employerContribution.contribution)
     }
-    const price = pricing.calculateProductPrice(products.longTermDisability, employee, selectedOptions)
-
-    expect(price).to.equal(22.04)
+    expect(commuterParking).to.equal(175)
   })
+})
+describe('getEmployerContribution', () => {})
+it('returns a truncated price with two decimal places not rounded when given a number with multiple places', () => {})
 
-  it('throws an error on unknown product type', () => {
-    const unknownProduct = { type: 'vision' }
+describe('calculatesLTDPrice', () => {
+      it('returns the LTD price minus the employer contribution', () => {
+        const selectedOptions = {
+          familyMembersToCover: [ee]
+        }
+        const price = pricing.calculateLTDPrice(products.voluntaryLife, employee, selectedOptions)
+      })
 
-    expect(() => pricing.calculateProductPrice(unknownProduct, {}, {})).to.throw('Unknown product type: vision')
-  })
-})
+      describe('calculateProductPrice', () => {
```

Comment:
Missing the end of your describe block for `calculateLTDPrice` so now this describe is inside that one. These details matter you gotta pay closer attention to them, it's what led to the 70% on the Migrating the Data assignment.